### DISCUSSION TITLE
Are there public instances of ArchiveBox?

### DISCUSSION THREAD

**Discussion by rwiesbach** [ArchiveBox/ArchiveBox#1491] [Q&A]
I understand that ArchiveBox is a software and that you do not provide a public instance yourself, but maybe there are public instances that you know of and that may be useful to others to list in this Q&A for ArchiveBox. (besides the [demo](https://demo.archivebox.io/) that seems to be dysfunctional right now "Error 1033 - Argo Tunnel error")

**Comment by pirate** [comment]
I'm currently working on changing the demo server setup a bit for the new v0.8.2 release, should be back up later this week. 

I have seen other public instances over the years but I don't want to share them without the owners permission, as they're not necessarily making them public for anyone to use for any purpose, they're usually aimed at their own specific communities needs.

**Comment by bluebrad** [comment]
looks like this would need to be the option so archive.org will have a decentralized option.

**Reply by pirate** [reply]
I'd love to provide something like that but running a public archive is a big deal. You have to be a fully-fledged non-profit acting as a "library" with $millions for laywers because you end up hosting a lot of copyrighted content. Archive.org only exists because Brewster was independently wealthy before starting it and could afford to lose hundreds of millions getting it going.

I could do it overnight from a technical perspective but it could jeapordize the whole project if I don't get the legal stuff right, and that takes $$$.

If you know good investors in the space who understand the value of internet archiving, let me know :)

I don't want to become a typical scraping company, and that's where a lot of VCs tend to push this sort of tech.

**Comment by bluebrad** [comment]
I see. TY for helping understand more .
Maybe working on a decentralized app that could work on NAS systems like
Synology and others.
i did get a docker version running but still have a bug where i can only
have one user. and do not have access to the files without the frontend
